docs/reference/modules/indices/circuit_breaker.asciidoc

```diff
@@ -115,3 +115,13 @@ documentation for more information.
     Limit for the number of unique dynamic scripts within a certain interval
     that are allowed to be compiled. Defaults to 75/5m, meaning 75 every 5 
     minutes.
+
+==== Operating Limits of Elasticsearch Clusters
```

Comment:
We try to use sentence case for headings. The `{es}` attribute expands to `Elasticsearch`. It helps us stay DRY and future proof.
```suggestion
==== Operating limits of {es} clusters
```